sechub-pds-examples/gosec/docker/GoSec-Ubuntu.dockerfile

```diff
@@ -6,15 +6,27 @@ FROM ${BASE_IMAGE}
 
 # The remaining arguments need to be placed after the `FROM`
 # See: https://ryandaniels.ca/blog/docker-dockerfile-arg-from-arg-trouble/
-ARG WORKSPACE="/workspace"
+
+# Folders
 ARG PDS_FOLDER="/pds"
-ARG PDS_CHECKSUM="f3101a6a2a96b39edf493249f2300c627fa01fcb7a6b96c5ae1edfbe70a31d46"
-ARG TOOL_FOLDER="/tool"
+ARG SCRIPT_FOLDER="/scripts"
+ENV TOOL_FOLDER="/tools"
+ARG WORKSPACE="/workspace"
+
+# PDS
+ENV PDS_VERSION=0.23.0
```

Comment:
I think we cannot use 0.23.0 but there is a need for a 0.23.1

see https://github.com/Daimler/sechub/issues/773
This is not really an issue here, but maybe a need to wait for the update.

Reply:
I would like to merge it and later we can create a new release of the PDS and integrate it.

Reply:
Just for documentaion: PDS 0.23.1 is now used.